### ISSUE TITLE
Proco loss error in code

### ISSUE THREAD

**Issue by range428** [LeapLabTHU/ProCo#3]
I have some questions about the implementation of `ProCo loss` in the code. 

Why does the ProCoLoss class return the `contrast_logits` value? And then it is fed into `LogitAdjust` together with `targets` to calculate the `scl_loss`? This does not match the Eq.20 in the paper.
**main.py**
```
contrast_logits1 = criterion_scl(f2, targets, args=args)
contrast_logits2 = criterion_scl(f3, targets, args=args)
     
scl_loss = (criterion_ce(contrast_logits1, targets) + criterion_ce(contrast_logits2, targets))/2 
```
**proco.py**
```
contrast_logits = LogRatioC.apply(kappa_new, torch.tensor(self.estimator.feature_num), logc)

return contrast_logits
```
And, how is the latter term in Eq,20 implemented?
![image](https://github.com/user-attachments/assets/e5724341-d3af-4371-b3fc-5192036c2901)
